yuyo/repeaters.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+# cython: language_level=3
+# BSD 3-Clause License
+#
+# Copyright (c) 2020-2021, crazygmr101
```

Comment:
Not sure about licensing rn but this should prob match the license for the whole project